src/components/Stations/StationsPage.tsx

```diff
@@ -1,16 +1,26 @@
-import { useState, useEffect } from "react";
+import { useAppStore } from "@/hooks/useAppStore";
```

Comment:
General proposal for this page.
My original idea regarding this page was to give the user a general outlook of all the stations. Maybe, we could add a fav stations section on top of the table, with all the fav stations there and the rest below. Just shooting ideas here.